sailfish-browser-captiveportal.desktop

```diff
@@ -5,5 +5,9 @@ Name=Captive Portal Browser
 X-MeeGo-Logical-Id=sailfish-browser-ap-name
 X-MeeGo-Translation-Catalog=sailfish-browser-captiveportal
 Icon=icon-launcher-browser
-Exec=invoker -s --type=browser -G /usr/bin/sailfish-browser-captiveportal -captiveportal %U
-Comment=Sailfish UI application
+Exec=/usr/bin/sailjail -p sailfish-browser-captiveportal.desktop /usr/bin/sailfish-browser-captiveportal -captiveportal %U
+Comment=Sailfish captive portal
+[X-Sailjail]
+Permissions=CaptivePortal;Privileged;Audio;Location;Internet;Downloads
+OrganizationName=org.sailfishos
+ApplicationName=browser.captiveportal
```

Comment:
Originally I envisioned that ApplicationName can't have a dot. I hoped dashes and underscores would suffice. Well, if it works...

Reply:
Me too.

Reply:
why didn't I just have "captiveportal" there's no need for this "browser.". You're right.